src/structures/paging/mapper/mapped_page_table.rs

```diff
@@ -140,6 +140,111 @@ impl<'a, P: PageTableFrameMapping> MappedPageTable<'a, P> {
 
         Ok(MapperFlush::new(page))
     }
+
+    /// Remove all empty P1-P3 tables
+    /// ## Safety
```

Comment:
Formatting nit, here and elsewhere, put newlines before and after `##Safety` for consistancy.